GameDesign/Assets/Scripts/TaserController.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TaserController : MonoBehaviour{
```

Comment:
Can we change it to `TaserManager` to be aligned with other classes?

Reply:
**_### No_**